pkg/codegen/nodejs/gen.go

```diff
@@ -2198,8 +2203,9 @@ func genNPMPackageMetadata(pkg *schema.Package, info NodePackageInfo) string {
 	if info.TypeScriptVersion != "" {
 		devDependencies["typescript"] = info.TypeScriptVersion
 	} else {
-		devDependencies["typescript"] = "^4.3.5"
+		devDependencies["typescript"] = MinumumTypescriptVersion
 	}
+	devDependencies["@types/node"] = MinumumNodeTypesVersion
```

Comment:
If it does exist in `info.DevDependencies`, it will be overwritten [here](https://github.com/pulumi/pulumi/blob/5fa6b0f72368b4734bc7cd2c343c1079232b4669/pkg/codegen/nodejs/gen.go#L2262-L2267).

Reply:
Ah, yes. 👍